Sources/XiEditor/Search.swift

```diff
@@ -388,59 +388,37 @@ extension EditViewController {
 
     func findStatus(status: [[String: AnyObject]]) {
         var findMarker: [Marker] = []
+        let statusStructs = status.flatMap(FindStatus.init)
```

Comment:
I think it would be good to know, somehow, if there is malformed data in here; that would indicate a bug in core.

Reply:
Do you mean an _integration_ test? 😄

Reply:
now that's an idea 🤔